src/Parsers/ASTSetQuery.h

```diff
@@ -13,6 +13,7 @@ class ASTSetQuery : public IAST
 {
 public:
     bool is_standalone = true; /// If false, this AST is a part of another query, such as SELECT.
+    bool is_clone = false; /// If true, this AST is a clone from other part of the query and should not be printed in format()
```

Comment:
I think the overall semantics would be more clear to the reader (e.g. in file "ASTSelectQuery.cpp") if this variable was named something like `bool print_in_format = true`.

Reply:
Sure. It makes sense to be more explicit :+1: